addons/pos_restaurant/static/src/js/Chrome.js

```diff
@@ -76,7 +76,8 @@ odoo.define('pos_restaurant.chrome', function (require) {
                 this.showScreen('FloorScreen', { floor: table ? table.floor : null });
             }
             _shouldResetIdleTimer() {
-                return super._shouldResetIdleTimer() && this.env.pos.config.iface_floorplan && this.mainScreen.name !== 'FloorScreen';
+                const can_close_payment = this.mainScreen.name === 'PaymentScreen' && this.env.pos.get_order().paymentlines.length === 0;
+                return can_close_payment && super._shouldResetIdleTimer() && this.env.pos.config.iface_floorplan && this.mainScreen.name !== 'FloorScreen';
```

Comment:
This means that the jump to FloorScreen may only happen when we're in PaymentScreen with 0 payment lines and the other scenarios are not taken into account anymore
You might prefer doing this: 
```suggestion
                const stayPaymentScreen = this.mainScreen.name === 'PaymentScreen' && this.env.pos.get_order().paymentlines.length > 0;
                return super._shouldResetIdleTimer() && !stayPaymentScreen && this.mainScreen.name !== 'FloorScreen';
```
`this.env.pos.config.iface_floorplan` can be removed from the condition because `_setActivityListeners()` already has this condition